commander/src/bootstrapping/commands/transaction/sign.ts

```diff
@@ -134,40 +126,33 @@ const signTransactionOffline = async (
 	metadata: ModuleMetadataJSON[],
 	transactionHexStr: string,
 ): Promise<Record<string, unknown>> => {
-	let signedTransaction: Record<string, unknown>;
+	const keys: Keys = {
+		mandatoryKeys: [],
+		optionalKeys: [],
+	};
 
-	if (!flags['include-sender'] && !flags['sender-public-key']) {
-		signedTransaction = await signTransaction(
-			flags,
-			registeredSchema,
-			metadata,
-			transactionHexStr,
-			flags['chain-id'],
-			{} as Keys,
-		);
-		return signedTransaction;
-	}
+	if (flags['sender-public-key']) {
+		const mandatoryKeys = flags['mandatory-keys'];
+		const optionalKeys = flags['optional-keys'];
+		if (!mandatoryKeys?.length && !optionalKeys?.length) {
+			throw new Error(
+				'--mandatory-keys or --optional-keys flag must be specified to sign transaction from multi signature account.',
+			);
```

Comment:
That was the existing code I just moved while simplifying the method flow, so it shows as a diff. Nevertheless,since we're here, it is definitely better to use oclif's built-in functionality for validating the flags, instead adding custom logic to our code :slightly_smiling_face:

I don't think that `dependsOn` option can handle situation when at least 1 of multiple flags is required, so I implemented it using `relationships` option.